config/crd/bases/keda.sh_clustertriggerauthentications.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.13.0
+    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
Undo this change please (just updating the tooling, remove your local `{workspace}/bin` folder and generate the manifests again)